SeqGAN/dataloader.py

```diff
@@ -1,10 +1,12 @@
 import numpy as np
 
+np.random.seed(88)
```

Comment:
You probably don't want these seed statements scattered across multiple files... it'll make it quite difficult to disable to random seed when you eventually want randomness. Just have one set at the top-level main.py (in this case, module_sequence_gan.py).

Reply:
Got it!